hack/get-metering-package-images.sh

```diff
@@ -2,4 +2,6 @@
 
 # kubectl -n openshift-marketplace port-forward svc/art-applications 50051
 
-grpcurl -plaintext -d '{"pkgName": "metering-ocp", "channelName": "4.2"}' localhost:50051 api.Registry/GetBundleForChannel  | jq '.csvJson' -r | jq '.spec.install.spec.deployments[0].spec.template.spec.containers[] | select(.name=="operator") | .env | map(select(.name | match(".*_IMAGE"))) | map(.value |= (. | ltrimstr("image-registry.openshift-image-registry.svc:5000/"))) | .[] | "export " + .name + "=" + .value' -r
+CHANNEL="${CHANNEL:-4.3}"
+
+grpcurl -plaintext -d '{"pkgName": "metering-ocp", "channelName": "'"$CHANNEL"'"}' localhost:50051 api.Registry/GetBundleForChannel  | jq '.csvJson' -r | jq '.spec.install.spec.deployments[0].spec.template.spec.containers[] | select(.name=="operator") | .env | map(select(.name | match(".*_IMAGE"))) | map(.value |= (. | ltrimstr("image-registry.openshift-image-registry.svc:5000/"))) | .[] | "export " + .name + "=" + .value' -r
```

Comment:
I guess we could also make the package name a variable too. Not sure how often that package name will be changed however.

Reply:
It shouldn't change, so I figured i would leave that out until we need that.